src/main/java/com/google/devtools/build/lib/remote/GrpcCacheClient.java

```diff
@@ -329,16 +330,23 @@ public ListenableFuture<Void> uploadActionResult(
 
   @Override
   public ListenableFuture<Void> downloadBlob(
-      RemoteActionExecutionContext context, Digest digest, OutputStream out) {
+      RemoteActionExecutionContext context,
+      Digest digest,
+      @Nullable DigestFunction.Value digestFunction,
```

Comment:
Out of curiosity, why would we permit this to be null?

Reply:
GrpcCacheClient implement RemoteCacheClient. For RemoteCacheClient, digestFunction could be null.

This is my first time to write java. Please point out my problems if you have any misunderstanding.